include/battle/battle.h

```diff
@@ -108,13 +108,13 @@ typedef struct SelfTurnData {
     int shellBellDamage;
 } SelfTurnData;
 
-typedef struct UnkBtlCtxSub_76 {
+typedef struct TrainerAIData {
```

Comment:
It seems that fields unk0 though unk18 inclusive are treated special in `ov10_0221BE20`.

Reply:
My guess is they're just everything that gets reset at the beginning of each turn